### DISCUSSION TITLE
Buzz* a chrome extension softphone

### DISCUSSION THREAD

**Discussion by grahambrown11** [l3nz/ari4java#170] [General]
# Buzz*

I thought share that I created a Chrome Extension softphone using WebRTC and SIP over WebSockets compatible with Asterisk

[![Get](https://raw.githubusercontent.com/grahambrown11/buzz-star/master/chrome_web_store.png)](https://chrome.google.com/webstore/detail/buzz-for-google-chrome/hkpciegbnpmpbpodnloafnoinjpbmakp?hl=en-GB)

**Comment by modrljin** [comment]
@grahambrown11 congratulations! I will try it for sure when I found time for it.

I will then share a link of what I did with my team recently. Check out web site [CTEngine](https://www.ctengine.net). In short words it is a platform for developing telephony and communication services. There we have shown 4 demo services: CTRouter, Web Call Center, Click-2-Call and Alarm!.

**Of course it is using Ari4Java!**
